src/module/updateTask.js

```diff
@@ -0,0 +1,14 @@
+import tasks from './taskStore.js';
+
+const updateTask = (index, description) => {
+  tasks.filter((object) => object.index === index).forEach((object) => {
+    object.description = description;
+  });
+};
+
+const completeTask = (index, completed) => {
+  tasks.filter((object) => object.index === index).forEach((object) => {
+    object.completed = completed;
```

Comment:
According to the requirement completed should set to Boolean value

Reply:
- Please check on the implementation part in  index.js file it accept only Boolean value